### PR TITLE
Webpack asset deployment

### LINKED ISSUE #156: Update deployment scripts to support npm/webpack bundled js

### PR DESCRIPTION
**Pull request by frgtn**
Wasn't able to fully test the procedure due to #159 so this will require another pass once that's fixed.

Fixes #156.

### CODE REVIEW

File: deployment/deploy.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 - hosts: all
-  sudo: yes
+  become: yes
+  become_method: sudo
```
Comment (sirex) [inline deployment/deploy.yml]: As far as I know there parameters requires a newer Ansible version, in this case, it would be at least useful to know, what Ansible version was using you or what minimal Ansible version is required.

Since Ansible evolves quite fast, some times I add this check:
https://github.com/python-dirbtuves/akl.lt/blob/master/deployment/deploy.yml#L14-L15

**Comment by sirex** [comment]
@frgtn thanks, great work! I did not tested it, but looks good to me.

File: package.json [hunk 2]
```diff
@@ -34,8 +34,8 @@
     "postcss-import": "^8.1.0",
     "postcss-loader": "^0.9.1",
     "precss": "^1.4.0",
-    "react": "^15.1.0",
-    "react-dom": "^15.1.0",
+    "react": "15.1.0",
```
Comment (chompomonim) [inline package.json]: Why you don't wanna use `^`? I'm ok with newer react versions.

File: deployment/deploy.yml (hunk 1 above)
Comment (frgtn) [inline deployment/deploy.yml]: Good point, I'll add a check.

There's not much of a reason for bumping minimal Ansible version except that I used my system Ansible which was recently upgraded to `2.1` in homebrew and was too lazy to setup a virtualenv to test with `1.x`.

File: package.json (hunk 2 above)
Comment (frgtn) [inline package.json]: This is an unrelated change I accidentally snuck in. I'm totally fine with not pinning the last version bit.
